samples/apps/copilot-chat-app/webapp/src/libs/useContentModerator.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { useMsal } from '@azure/msal-react';
+import { useAppDispatch, useAppSelector } from '../redux/app/hooks';
+import { RootState } from '../redux/app/store';
+import { FeatureKeys } from '../redux/features/app/AppState';
+import { addAlert, toggleFeatureState } from '../redux/features/app/appSlice';
+import { AuthHelper } from './auth/AuthHelper';
+import { AlertType } from './models/AlertType';
+import { ContentModerationService } from './services/ContentModerationService';
+
+const riskThreshold = 4;
```

Comment:
can we add a comment similar to what you added in `AzureContentModerator` to explain where this # comes from?